src/Forms/FieldList.php

```diff
@@ -180,7 +180,7 @@ protected function flushFieldsCache()
     }
 
     /**
-     * @deprecated 4.1.0:5.0.0 Please use dataFields or saveableFields
+     * @deprecated 4.1.0 Please use dataFields or saveableFields
```

Comment:
```suggestion
     * @deprecated 4.1.0 Please use dataFields or saveableFields instead
```
Also update the version and message in the notice below.

Reply:
Updated